pkg/server/server.go

```diff
@@ -282,8 +282,11 @@ func writeKubeConfig(certs string, config *Config) error {
 	}
 
 	if err = clientaccess.WriteClientKubeConfig(kubeConfig, url, config.ControlConfig.Runtime.ServerCA, config.ControlConfig.Runtime.ClientAdminCert,
-		config.ControlConfig.Runtime.ClientAdminKey); err != nil {
+		config.ControlConfig.Runtime.ClientAdminKey); err == nil {
+		logrus.Infof("Wrote kubeconfig %s", kubeConfig)
```

Comment:
Do we want the contents leaking here?

Reply:
`kubeconfig` contains the path to the file, not the contents.

Reply:
Sounds good. As a non blocking thing then, it'd probably be good to update the variable name to represent the contents a more clearly.

Reply:
Are you comfortable approving as-is @briandowns?